src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -708,7 +714,7 @@
 										<Setter Target="PlaceholderElement.Foreground" Value="{ThemeResource OutlinedTextBoxPlaceholderForegroundFocused}" />
 										<Setter Target="HeaderElement.Foreground" Value="{ThemeResource OutlinedTextBoxHeaderForegroundFocused}" />
 										<Setter Target="RootBorder.BorderThickness" Value="{ThemeResource MaterialOutlinedTextBoxBorderThicknessFocused}" />
-										<Setter Target="RootBorder.Padding" Value="0" />
+										<Setter Target="RootBorder.Padding" Value="{ThemeResource MaterialOutlinedTextBoxPaddingFocused}" />
```

Comment:
@kazo0, @eriklimakc, @Xiaoy312 please help me remember if there is a particular reason why we have a mismatch of "OutlinedTextBox..." and "MaterialOutlinedTextBox..." for the ThemeResources for example here please

Reply:
not that i remember of, other than negligence

Reply:
It seems that from the very start of the lightweight style, this was already like that. You can see it [here](https://github.com/unoplatform/Uno.Themes/commit/618fc2e44209cc3967150ce4a6e601a1bbd5c200#diff-f6c7c34fda0976ee4fb202c4d86552ca497e13872007211455fa4aa161533995).

Reply:
I don't think there is a reason for that, not that I recall

Reply:
@kazo0, @Xiaoy312, @eriklimakc We have all these resources below starting with "Material" that we will need to review I think then. Just wanted to make sure I did not forget a specific reason why it was kept "Material" in the name.
And I imagine it will be a breaking change as well for the resource's naming with the documentation update.

- https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/TextBox.md
- https://github.com/unoplatform/Uno.Themes/blob/c35a540d0dfe557519603a4fad3b38e1c72ba7df/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L149-L168

Seems we would need to double-check other controls also (maybe on separate PRs for other controls ? so it is easier to follow the adjustments):
https://github.com/search?q=repo%3Aunoplatform%2FUno.Themes+%22%7C+%60Material%22&type=code